pkg/tsdb/influxdb/flux/macros.go

```diff
@@ -0,0 +1,45 @@
+package flux
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+)
+
+const variableFilter = `(?m)([a-zA-Z]+)\.([a-zA-Z]+)`
+
+// Interpolate processes macros
+func Interpolate(query QueryModel) (string, error) {
```

Comment:
We should also support regular template variables here. I admit this broke some backwards compatibility with the previous plugin where we had a predefined "$range" variable. One more though, these are just statically generated, but I wonder if we could query influx to ask for what query/template variables it supports?